src/apps/recommendation/recommendation.dev.tsx

```diff
@@ -0,0 +1,61 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import React from "react";
+import Recommendation, {
+  RecommendationEntryProps
+} from "./recommendation.entry";
+import globalTextArgs, {
+  GlobalEntryTextProps
+} from "../../core/storybook/globalTextArgs";
+import serviceUrlArgs from "../../core/storybook/serviceUrlArgs";
+import RecommendationSkeleton from "./RecommendationSkeleton";
+
+export default {
+  title: "Apps / Recommendation",
+  component: Recommendation,
+  argTypes: {
+    wid: {
+      defaultValue: "work-of:870970-basis:136336282",
+      control: { type: "text" }
+    },
+    positionImageRight: {
+      defaultValue: false,
+      control: { type: "boolean" }
+    },
+    materialUrl: {
+      defaultValue: "/work/:workid",
+      control: { type: "text" }
+    },
+    etAlText: {
+      name: "Et al. Text",
+      defaultValue: "et al.",
+      control: { type: "text" }
+    },
+    ...globalTextArgs,
+    ...serviceUrlArgs
+  }
+} as ComponentMeta<typeof Recommendation>;
+
+export const App: ComponentStory<typeof Recommendation> = (
+  args: RecommendationEntryProps & GlobalEntryTextProps
+) => <Recommendation {...args} />;
+
+App.argTypes = {
+  positionImageRight: {
+    defaultValue: false,
+    control: { type: "boolean" }
+  }
+};
```

Comment:
Please consider if this is necessary.

As far as I can tell this is same setup as on the default export with `defaultValue: false` etc. so I wonder why it has been added.

Reply:
True, missed this.